src/main/java/training/supportbank/Main.java

```diff
@@ -1,23 +1,39 @@
 package training.supportbank;
-import java.io.BufferedReader;
-import java.io.FileReader;
-import java.io.IOException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import java.math.BigDecimal;
 import java.util.ArrayList;
 import java.util.Scanner;
 
 
 public class Main {
+    private static final Logger LOGGER = LogManager.getLogger();
     public static void main(String args[]) {
-        String CSV = "/Users/kbolourc/Training/bootcampPersonalGitHub/Week1/SupportBank-Java-Template/Transactions2014.csv";
-        ArrayList<Account> listOfAccounts = readCSV(CSV);
+        String path = args[0];
+        ArrayList<Account> listOfAccounts;
+
+        switch (path.split("\\.")[1]) {
+            case "csv":
+                listOfAccounts = AccountCreate.readCSV(path);
+                break;
+            case "json":
+                listOfAccounts = AccountCreate.readJSON(path);
+                break;
+            case "xml":
+                listOfAccounts = AccountCreate.readXML(path);
+                break;
+            default:
+                LOGGER.fatal("Incorrect filetype");
+                return;

```

Comment:
Really elegant having a switch here, it makes your code really easy to understand and follow